refinery/ui/source/js/commons/treemap/template.html

```diff
@@ -50,7 +50,8 @@
         <svg class="icon-svg">
           <use
             xmlns:xlink="http://www.w3.org/1999/xlink"
-            xlink:href="/static/images/icons.svg#maximize"></use>
+            xlink:href=""
```

Comment:
Why is this needed?

Reply:
Not sure, but it it doesn't work without it. 
I followed the advice from [here](https://stackoverflow.com/a/25609356/6031066).